optimum/intel/openvino/modeling_visual_language.py

```diff
@@ -0,0 +1,1192 @@
+        local_files_only: bool = False,
+        load_in_8bit: bool = False,
+        quantization_config: Union[OVWeightQuantizationConfig, Dict] = None,
+        **kwargs,
+    ):
+        """
+        Loads a model and its configuration file from a directory or the HF Hub.
+
+        Arguments:
+            model_id (`str` or `Path`):
+                The directory from which to load the model.
+                Can be either:
+                    - The model id of a pretrained model hosted inside a model repo on huggingface.co.
+                    - The path to a directory containing the model weights.
+            use_auth_token (Optional[Union[bool, str]], defaults to `None`):
+                Deprecated. Please use `token` instead.
+            token (Optional[Union[bool, str]], defaults to `None`):
+                The token to use as HTTP bearer authorization for remote files. If `True`, will use the token generated
+                when running `huggingface-cli login` (stored in `~/.huggingface`).
+            revision (`str`):
+                The specific model version to use. It can be a branch name, a tag name, or a commit id.
+            force_download (`bool`, *optional*, defaults to `False`):
+                Whether or not to force the (re-)download of the model weights and configuration files, overriding the
+                cached versions if they exist.
+            cache_dir (`Union[str, Path]`, *optional*):
+                The path to a directory in which a downloaded pretrained model configuration should be cached if the
+                standard cache should not be used.
+            encoder_file_name(`str`, *optional*):
+                The encoder model file name. Overwrites the default file name openvino_encoder_model.xml and allows one to
+                load the encoder model with a different name.
+            decoder_file_name(`str`, *optional*):
+                The decoder model file name. Overwrites the default file name openvino_decoder_model.xml and allows one to
+                load the decoder model with a different name.
+            decoder_with_past_file_name(`str`, *optional*):
+                The decoder with past key values model file name overwriting the default file name
+                openvino_decoder_with_past_model.xml, allowing to load the decoder model with a different name.
+            local_files_only(`bool`, *optional*, defaults to `False`):
+                Whether or not to only look at local files (i.e., do not try to download the model).
+        """
+        if use_auth_token is not None:
+            warnings.warn(
+                "The `use_auth_token` argument is deprecated and will be removed soon. Please use the `token` argument instead.",
+                FutureWarning,
+            )
+            if token is not None:
+                raise ValueError("You cannot use both `use_auth_token` and `token` arguments at the same time.")
+            token = use_auth_token
```

Comment:
shouldn't be needed I think as already check in https://github.com/huggingface/optimum-intel/blob/6a8718b76a5fc5e6796483746eded64e4007e94d/optimum/intel/openvino/modeling_base.py#L394

```suggestion
```

Reply:
removed